runtimes/runtimes/standalone.test.ts

```diff
@@ -102,6 +113,7 @@ describe('standalone', () => {
                 encryptionInitialization.key,
                 encryptionInitialization.mode
             )
+            await setTimeout(() => Promise.resolve(), 1000)
```

Comment:
Is this 1s timeout here necessary? If so, why?

Reply:
yes for this specific test case - there is an async call inside an async call. If I don't add the timeout, the function after this line is executed later - so to avoid that situation I have added this timeout.